### PR TITLE
add support for molten chain

### PR DESCRIPTION
**Pull request by dshiell**
## Summary by CodeRabbit

- **New Features**
	- Added `MoltenIcon` functionality for rendering a molten effect icon in the app.
	- Updated the `Chain` interface to include an optional `size` parameter for icons.

### CODE REVIEW

**Comment by dshiell** [comment]
sorry for the reformatting, need to get my formatter in sync with however you like @mvpoyatt

File: app/utils/chains/icons.tsx [hunk 1]
```diff
@@ -18,3 +18,14 @@ export const BaseIcon = (): JSX.Element => {
     </svg>
   );
 };
+
+// TODO: fix the molten icon
+export const MoltenIcon = (): JSX.Element => {
+  return (
+    <svg className="mr-1.5" width="30" height="30" viewBox="0 0 111 111" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M54.921 110.034C85.359 110.034 110.034 85.402 110.034 55.017C110.034 24.6319 85.359 0 54.921 0C26.0432 0 2.35281 22.1714 0 50.3923H72.8467V59.6416H3.9565e-07C2.35281 87.8625 26.0432 110.034 54.921 110.034Z"
```
Comment (dshiell) [inline app/utils/chains/icons.tsx:27]: Currently copied from base

File: .env.production [hunk 2]
```diff
@@ -0,0 +1,5 @@
+API_URL=http://localhost:26657
```
Comment (dshiell) [inline .env.production:1]: todo: remove

File: .env.staging [hunk 3]
```diff
@@ -0,0 +1,7 @@
+API_URL=http://localhost:26657
```
Comment (dshiell) [inline .env.staging:1]: todo: remove

**Review by Inkvi — APPROVED** [review]

**Pull request merged**